extensions/narayana-lra/deployment/src/main/java/io/quarkus/narayana/lra/deployment/NarayanaLRAProcessor.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.narayana.lra.deployment;
+
+import static io.quarkus.deployment.annotations.ExecutionTime.RUNTIME_INIT;
+
+import org.jboss.jandex.DotName;
+
+import io.narayana.lra.client.internal.proxy.nonjaxrs.LRAParticipantRegistry;
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.arc.deployment.BeanDefiningAnnotationBuildItem;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
+import io.quarkus.narayana.lra.runtime.LRAConfiguration;
+import io.quarkus.narayana.lra.runtime.NarayanaLRAProducers;
+import io.quarkus.narayana.lra.runtime.NarayanaLRARecorder;
+
```

Comment:
I think this class should set the Feature. Like feature.produce(new FeatureBuildItem(Feature.NARAYANA_LRA));

Reply:
okay will do

Reply:
Seems to be done in the registerFeature method.

Reply:
Yes, I made changes after the first round of reviews.